packages/zoe/src/zoeService/zoe.js

```diff
@@ -243,6 +218,68 @@ function makeZoe(vatAdminSvc, zcfBundleName = undefined) {
             zoeSeatAdmins.forEach(zoeSeatAdmin => zoeSeatAdmin.fail(reason));
           },
           stopAcceptingOffers: () => (acceptingOffers = false),
+          makeUserSeat: async (
+            invitationHandle,
+            initialAllocation,
+            proposal,
+          ) => {
+            const offerResultPromiseKit = makePromiseKit();
+            // Don't trigger Node.js's UnhandledPromiseRejectionWarning.
+            // This does not suppress any error messages.
+            offerResultPromiseKit.promise.catch(_ => {});
+            const exitObjPromiseKit = makePromiseKit();
+            // Don't trigger Node.js's UnhandledPromiseRejectionWarning.
+            // This does not suppress any error messages.
+            exitObjPromiseKit.promise.catch(_ => {});
+            const seatHandle = makeHandle('SeatHandle');
+
+            const { userSeat, notifier, zoeSeatAdmin } = makeZoeSeatAdminKit(
+              initialAllocation,
+              exitZoeSeatAdmin,
+              hasExited,
+              proposal,
+              withdrawPayments,
+              exitObjPromiseKit.promise,
+              offerResultPromiseKit.promise,
+            );
+
+            seatHandleToZoeSeatAdmin.init(seatHandle, zoeSeatAdmin);
+
+            const seatData = harden({ proposal, initialAllocation, notifier });
+
+            zoeSeatAdmins.add(zoeSeatAdmin);
+
+            E(/** @type {Promise<AddSeatObj>} */ (addSeatObjPromiseKit.promise))
+              .addSeat(invitationHandle, zoeSeatAdmin, seatData, seatHandle)
+              .then(({ offerResultP, exitObj }) => {
+                offerResultPromiseKit.resolve(offerResultP);
+                exitObjPromiseKit.resolve(exitObj);
+              })
+              // Don't trigger Node.js's UnhandledPromiseRejectionWarning.
+              // This does not suppress any error messages.
+              .catch(() => {});
```

Comment:
I understand the other ones. But for this one, why is it that it is safe to ignore a rejection if it happens? Here, we would be silently dropping on the floor either an error from `addSeat` or an error from the `then` callback. In that case, suppressing Node's diagnostic actually masks those errors.

Reply:
Ah, you are correct. I think this is the same as the old code in this regard. I'll drop the catch.